<?xml version="1.0"?>
<launch>
    <arg name="namespace" default="auv324"/>
    <arg name="with_ff" default="0"/>
    <arg name="x_type" default="1"/>
    <arg name="debug" default="0" />

    <node name="auv_traj_follow_manager_node" pkg="auv_controller" type="auv_traj_follow_manager_node" 
        respawn="false" output="screen" args="$(arg namespace) $(arg with_ff) $(arg x_type) $(arg debug)">
        <rosparam param="target_waypoint">"[[20.0, 0.0], [100.0, 10.0], [150.0, 15.0]]"</rosparam>
        <rosparam param="desired_x_linear_velocity">1.5432</rosparam>
        <rosparam param="traj_follow_manage_period">0.2</rosparam>
        <rosparam param="threshold">2.5</rosparam>
    </node>
</launch>
